src/scss/_main.scss

```diff
@@ -241,3 +241,98 @@ section {
 .arc text {
   font-size: 10px;
 }
+
+.node-panel {
+  padding: 0px !important;
+  padding-top: 5px !important;
+  margin-bottom: 5px;
+}
+
+.node-circle-container {
+  display: flex;
+  align-items: center;
+}
+
+.node-circle {
+  width: 10px;
+  height: 10px;
+  border-radius: 100%;
+  padding: 0 !important;
+  margin-right: 10px;
+}
+
+.node-circle.blue {
+  background-color: #2196f3;
+}
+
+.node-circle.orange {
+  background-color: orange;
+}
+
+.node-circle.red {
+  background-color: red;
+}
+
+.node-barchart-container {
+  display: flex;
+  justify-content:flex-end;
+  align-items: center;
+}
+
+.node-barchart-container .domain {
+  display: none;
+  visibility: hidden;
+}
+
+.node-barchart {
+  visibility: visible;
+  opacity: 1;
+  transition: all ease-in 1s;
+}
+
+.node-hovered-container {
+  display: flex;
+  align-items: center;
+  visibility: hidden;
+  opacity: 0;
+  z-index: 10;
+  background-color: white;
+  height: 25px;
+  right: 100px;
+  padding: 5px;
+  position: absolute;
+  transition: all ease-out 1s;
```

Comment:
Let's change every animation speed here to something faster. `0.1s` should be good.